web/src/features/panels/zone/ZoneDetails.tsx

```diff
@@ -14,6 +14,7 @@ import {
   spatialAggregateAtom,
   timeAverageAtom,
 } from 'utils/state/atoms';
+import { useBreakpoint } from 'utils/styling';
```

Comment:
Unrelated to this PR but I think I am going to move this to the hooks folder. It makes no sense it's in `utils/styling` when we have a folder dedicated to hooks.

Reply:
I agree